src/cgame/cg_event.cpp

```diff
@@ -121,7 +121,7 @@ static void CG_Obituary( entityState_t *ent )
 
 	target = ent->otherEntityNum;
 	attacker = ent->otherEntityNum2;
-	assistant = ent->otherEntityNum3;
+	assistant = ent->groundEntityNum; // we hijack the field for this
```

Comment:
What's the point of this change?

Reply:
It makes it more obvious that the field is hijacked, and it will allow dropping the define later. Note the commit is extracted from https://github.com/Unvanquished/Unvanquished/pull/2312.